roles/nf-core/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+nextflow_java: "/sw/comp/java/x86_64/sun_jdk1.8.0_151"
+nextflow_version_tag: "v19.10.0"
```

Comment:
A new version of nextflow came out a few days ago, so you can grab that now

Reply:
Done!